CMU-313/nodebb-f24-team-kale
# Reply anonymously

**Description:**
I implemented a new feature in the Quick Reply module that allows users to post replies anonymously. The feature provides users with an option to reply to topics without revealing their identity.

**I made the following changes:**
1. Anonymous Reply Button:
- Added a "Reply Anonymously" button next to the existing Quick Reply button in the UI.
- Attached a click event handler to this new button that calls the postReply function with the anonymous flag set to true.
2. Refactored postReply Function:
- Updated the postReply function to accept an additional anonymous parameter.
- Modified the API call to include the anonymous flag in the request data, allowing the backend to handle anonymous replies.
![Screenshot 2024-09-29 at 2 20 35 PM](https://github.com/user-attachments/assets/551bd0ca-b2df-4008-82b5-5211719a3a14)

**Overview:** The new "Reply Anonymously" button is visible alongside the existing Quick Reply button which improves students' privacy and encourages more open discussions by providing an option for anonymous participation.

**How to Test:**
- Navigate to any discussion thread
- You can see a "Reply Anonymously" button.

**Resolves:**  #20
